### PR TITLE
[feat] Add description to text explore table

### PR DESCRIPTION
**Pull request by VkoHov**
1. Added `Description` column to text explore table
2. Change `Run` to `Run Name `
3. Added `Date` column to text explore table

### CODE REVIEW

**Review by rubenaprikyan — APPROVED** [review]
looks good

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 ### Enhancements:
 - Hugging Face adapter refactoring (mihran113)
 
+## 3.7.1 Mar 10, 2022
```
Comment (Hamik25) [inline CHANGELOG.md:7]: Please don't forget to change the version to 3.7.3 and date

File: aim/VERSION [hunk 2]
```diff
@@ -1 +1 @@
-3.7.0
+3.7.1
```
Comment (Hamik25) [inline aim/VERSION:1]: Need change version to 3.7.3

File: aim/web/ui/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ui_v2",
-  "version": "3.7.0",
+  "version": "3.7.1",
```
Comment (Hamik25) [inline aim/web/ui/package.json:3]: Need change version to 3.7.3

File: aim/web/ui/src/services/models/textExplorer/textExplorerAppModel.ts [hunk 4]
```diff
@@ -1080,7 +1080,9 @@ function getDataAsTableRows(
           color: metricsCollection.color ?? metric.color,
           dasharray: metricsCollection.dasharray ?? metric.dasharray,
           experiment: metric.run.props.experiment?.name ?? 'default',
-          run: moment(metric.run.props.creation_time * 1000).format(
+          run: metric.run.props?.name ?? '-',
+          description: metric.run.props?.description ?? '-',
+          date: moment(metric.run.props.creation_time * 1000).format(
             'HH:mm:ss · D MMM, YY',
```
Comment (Hamik25) [inline aim/web/ui/src/services/models/textExplorer/textExplorerAppModel.ts:1086]: Please move to `src/config/dates/dates.ts` this file. Then us it here.

**Review by Hamik25 — CHANGES_REQUESTED** [review]

**Comment by Hamik25** [comment]
Please also add a description for PR.

**Review by Hamik25 — APPROVED** [review]

**Pull request merged**